person.rb

```diff
@@ -1,9 +1,12 @@
+require_relative './decorator'
+require_relative './nameable'
 # Person class
 class Person
```

Comment:
- [ ] The `Person` class inherits from `Nameable` class.
```suggestion
class Person < Nameable
```